_dev/src/components/smart-shopping-campaign/smart-shopping-campaign-table-list-row.vue

```diff
@@ -93,12 +96,19 @@ export default {
     campaignCountryName() {
       return this.$options.filters.changeCountriesCodesToNames([this.campaign.targetCountry])[0];
     },
+    totalProducts() {
+      return retrieveProductNumberFromFiltersIds(
+        this.campaign.productFilters, this.$store.state.smartShoppingCampaigns.sscAvailableFilters,
+      );
+    },
     campaignProducts() {
       return this.campaign.productFilters?.length
         // eslint-disable-next-line
-        ? this.$i18n.tc('smartShoppingCampaignCreation.nbProductsSelected',
-          this.campaign.productFilters[0].values.length,
-          [this.campaign.productFilters[0].values.length])
+        ? this.$i18n.t('smartShoppingCampaignCreation.dimensionXFilterSelected',
+          [this.$i18n.t(`smartShoppingCampaignCreation.${this.campaign.productFilters[0].dimension}`)])
+          + ' ' + this.$i18n.tc('smartShoppingCampaignCreation.xProducts',
+          this.totalProducts,
+          [this.totalProducts])
```

Comment:
I think we can have a simpler way to write it

```suggestion
        ? this.$i18n.t(
            'smartShoppingCampaignCreation.dimensionXFilterSelectedWithXProducts',
            this.totalProducts,
            [
              this.$i18n.t(`smartShoppingCampaignCreation.${this.campaign.productFilters[0].dimension}`),
              this.totalProducts,
            ]
          )
```

With `dimensionXFilterSelectedWithXProducts` = "{0} filter | {0} filter ({1} product) | {0} filter ({1} products)"